horace_core/sqw/@DnDBase/copy.m

```diff
@@ -0,0 +1,7 @@
+function wout = copy(w)
+% Copy this DnD object or an array of DnD objects
+%
+%   >> wout = copy(w);
+
+wout = w;
```

Comment:
As DnDBase is a handle class, this should not perform a true copy. Apparently subclassing from Matlab.Mixin.Copyable should fix this